src/cmakeTools.ts

```diff
@@ -130,24 +130,24 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         return this._useCMakePresets;
     }
     async setUseCMakePresets(useCMakePresets: boolean) {
-        if (this.targetName === this._initTargetName) {
+        if (this.targetName.value === this.initTargetName) {
             if (useCMakePresets) {
-                this._targetName.set(this._targetsInPresetName);
+                this.targetName.set(this.targetsInPresetName);
             } else {
-                this._targetName.set('all');
+                this.targetName.set('all');
             }
         }
-        if (!useCMakePresets && this.targetName === this._targetsInPresetName) {
-            this._targetName.set('all');
+        if (!useCMakePresets && this.targetName.value === this.targetsInPresetName) {
+            this.targetName.set('all');
         }
         const oldValue = this._useCMakePresets;
```

Comment:
I noticed you changed `this.useCMakePresets` -> `this._useCMakePresets` everywhere.  Are we doing that with other `get`s that have a `_` variable as the backing storage?  After this formatting change we may want to go back and double check for consistency.

Reply:
no we are not doing it with other gets, I fixed this. I have to make sure it didnt happen on other places.